notify/wechat/wechat_test.go

```diff
@@ -26,6 +28,35 @@ import (
 	"github.com/prometheus/alertmanager/notify/test"
 )
 
+// TestNotify can test the entire notification process as long as you fill in
+// the correct configuration: secret and WechatConfig.
+// You can get a full description of the WechatConfig's fields on
+// https://work.weixin.qq.com/api/doc/90001/90143/91199
+func TestNotify(t *testing.T) {
```

Comment:
The unit test can't access the real WeChat endpoint. We need to mock the WeChat API to simulate the various responses. Since it isn't trivial, I can have a look at it. Also this code doesn't test anything.